Date: Jul 24, 2024
From: smuthu545 <Sindhuja_Muthukrishnan@comcast.com>
Source: COMCAST
Subject: 1. Heaptrack is upgraded to 1.5.0 to support runtime attachment.
2. Add the thread info change in latest version
3. remove zstd dependency in heaptrack (if zstd depends not added, 
kirkstone builds are failing. If zstd added, heaptrack output is not captured properly.
So added a patch to remove the zstd dependency in heaptrack bin)
Signed-off-by: smuthu545 <Sindhuja_Muthukrishnan@comcast.com>
Index: git/src/track/heaptrack.sh.cmake
===================================================================
--- git.orig/src/track/heaptrack.sh.cmake
+++ git/src/track/heaptrack.sh.cmake
@@ -283,11 +283,6 @@ output_suffix="gz"
 COMPRESSOR="gzip -c"
 UNCOMPRESSOR="gzip -dc"

-if [ "@ZSTD_FOUND@" = "TRUE" ] && [ ! -z "$(command -v zstd 2> /dev/null)" ]; then
-    output_suffix="zst"
-    COMPRESSOR="zstd -c"
-    UNCOMPRESSOR="zstd -dc"
-fi

 output_non_raw="$output.$output_suffix"
